### ISSUE TITLE
Is this solution working with Flutter?

### ISSUE THREAD

**Issue by giorgio79** [ananay/apple-auth#23]
Looking for a Flutter compatible Apple signin :) https://pub.dev/packages/sign_in_with_apple

Will this work?

**Comment by tuzimoe** [comment]
Yes it works. Our flutter team already successfully be able to use flutter Sign In With Apple with this repo and Koa as backend. Feel free to ask any questions
